lib/iex/lib/iex/introspection.ex

```diff
@@ -70,7 +72,11 @@ defmodule IEx.Introspection do
         h(mod, fun, arity)
       end
 
-      if result != [], do: :ok, else: :not_found
+      cond do
+        result != []            -> :ok
+        has_callback?(mod, fun) -> :behaviour_found
+        true                    -> :not_found
```

Comment:
I think we are avoiding the additional spaces to indent things. So maybe put them in a new line
```elixir
result != []
  -> :ok
```